transport/transport-runtime/src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,7 @@
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.google.android.datatransport.runtime">
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
You are missing a service declaration in the manifest, it will cause an exception at runtime.